packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -149,3 +152,24 @@ export function wrapSummaryInChannelsTree(summarizeResult: ISummaryTreeWithStats
     };
     summarizeResult.stats.treeNodeCount++;
 }
+
+export async function getFluidDataStoreAttributesFromStorageOrSnapshot(
```

Comment:
I'd simplify name. Usually we do not enumerate type of arguments in name of function (snapshot, storage).
maybe readFluidDataStoreAttributes?

Reply:
Shortened it to "getFluidDataStoreAttributes"